website/src/components/EventSummaryCollection/EventSummaryCollection.jsx

```diff
@@ -5,7 +5,7 @@ import './EventSummaryCollection.css';
 
 class EventSummaryCollection extends Component {
   static propTypes = {
-    event: PropTypes.array,
+    event: PropTypes.object,
```

Comment:
Quick attention, i tho it should be propTypes.shape {..}

Reply:
Thanks i did not see this